howfairis/checker.py

```diff
@@ -77,12 +74,8 @@ def remove_comments(text):
             else:
                 readme_file_format = None
 
-            if self.include_comments is True:
-                text = response.text
-            else:
-                text = remove_comments(response.text)
-
-            return Readme(filename=readme_filename, text=text, file_format=readme_file_format)
+            return Readme(filename=readme_filename, text=response.text, file_format=readme_file_format,
+                          ignore_commented_badges=not self.ignore_commented_badges)
```

Comment:
Huh, why invert ignore_commented_badges?
```suggestion
                          ignore_commented_badges=self.ignore_commented_badges)
```

When I set `ignore_commented_badges: false` the warning should not appear.
When I set `ignore_commented_badges: true` the warning should only appear when a README.rst is found.

Now it is the other way around.

Reply:
Simply a mistake on my part, I must have forgot to flip the sign after migrating from `self.include_comments`